M2/Macaulay2/packages/Polyhedra/extended/directProduct.m2

```diff
@@ -1,6 +1,6 @@
 
 -- PURPOSE : Computing the direct product of two polyhedra in the direct product of their ambient spaces
-directProduct = method()
+directProduct = method(Binary => true)

```

Comment:
Just a thought: `directProduct` is the kind of method name that perhaps we should put in `shared.m2`.

Reply:
Good call

Reply:
Done!